docs/source/reference/storages.rst

```diff
@@ -13,7 +13,7 @@ The :mod:`~optuna.storages` module defines a :class:`~optuna.storages.BaseStorag
    optuna.storages.RetryFailedTrialCallback
    optuna.storages.fail_stale_trials
    optuna.storages.JournalStorage
-   optuna.storages.JournalFileStorage
```

Comment:
If we remove this from this file, the explanation about previous classes disappear from document for users. How about leaving them as they are?

Reply:
Agreed.